### PR TITLE
set dubbo playload

### PR DESCRIPTION
**Pull request by lxd5866**
update SetData() method to set Frame.payload , the Frame.payload  is used by encode the frame

### CODE REVIEW

**Comment by cch123** [comment]
顺便补一下测试就更好了~

File: pkg/protocol/xprotocol/dubbo/command.go [hunk 1]
```diff
@@ -80,6 +80,7 @@ func (r *Frame) GetData() types.IoBuffer {
 
 func (r *Frame) SetData(data types.IoBuffer) {
 	r.content = data
+	r.payload = data.Bytes()
```
Comment (champly) [inline pkg/protocol/xprotocol/dubbo/command.go:83]: Should modify frame.DataLen too?

**Comment by champly** [comment]
Please add a unit test.

File: pkg/protocol/xprotocol/dubbo/command_test.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	}
+	content := frame.GetData().Bytes()
+	if !reflect.DeepEqual(content, baseContent) {
+		t.Errorf("dubbo method GetData error")
+	}
+	byteFrame, error := encodeFrame(nil, frame)
+	if error != nil || !reflect.DeepEqual(byteFrame.Bytes(), encodeData) {
+		t.Errorf("dubbo encode freame panic:%s", error)
+	}
+}
```
Comment (champly) [inline pkg/protocol/xprotocol/dubbo/command_test.go:52]: You can use `decodeFrame` decode byte, compare content.

**Comment by champly** [comment]
LGTM

**Comment by cch123** [comment]
LGTM

**Review by cch123 — APPROVED** [review]

**Review by champly — APPROVED** [review]

**Pull request merged**